### ISSUE TITLE
Crashes with MacOS builds

### ISSUE THREAD

**Issue by DedPimpin** [rlabrecque/Steamworks.NET#233]
Hi I'm having trouble with my MacOS build.  I have a multiplayer game and am successfully using Steamworks.net to grab things like the players names from steam, with my Windows build.  With the Steamworks.net plugin on my first screen, my game crashes instantly on Mac.  Any ideas what is happening?  Does this work on Mac typically?  I'm using Unity 2017.3.0f3

**Comment by rlabrecque** [comment]
It should work fine, do you have the log file?

**Comment by DedPimpin** [comment]
Hey, thanks for getting back.  I realized the problem was talked about in an older thread.  The Mac I was testing on was running Yosemite 10.10.5  and it sounds like others had issues with it as well.  I installed High Sierra and everything worked perfectly.  Was still wondering if there is a fix you would recommend for Yosemite users?  I know some were mentioned in the thread https://github.com/rlabrecque/Steamworks.NET/issues/205  such as disabling the Steam overlay for those users.... or just tell them to upgrade their OS lol.
